platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/task/LearnerPushTask.java

```diff
@@ -87,14 +87,16 @@ public LearnerPushTask(
             final Queue<MerkleNode> rootsToReceive,
             final AtomicReference<T> root,
             final LearnerTreeView<T> view,
-            final ReconnectNodeCount nodeCount) {
+            final ReconnectNodeCount nodeCount,
+            final ReconnectMapStats mapStats) {
```

Comment:
missing nullity annotation and javadoc for new param

Reply:
added